core/session.rb

```diff
@@ -65,6 +65,8 @@ class Session
   attr_reader :tool_config
   attr_accessor :show_help
   attr_accessor :reinstall
+  attr_accessor :recreate
```

Comment:
You must also set default value for the instance variable in the `initialize` method.